static/docs/style.css

```diff
@@ -1,5 +1,5 @@
 img {
-  max-width: 100%;
+  max-width: 1.8cm;
```

Comment:
This selector is too broad for this change. Some of our examples also include screenshots. Let's use .logo img or something.

Reply:
oh whoops, yes.